src/game_api/script.cpp

```diff
@@ -818,6 +818,8 @@ SpelunkyScript::ScriptImpl::ScriptImpl(std::string script, std::string file, Sou
     lua["entity_has_item_uid"] = entity_has_item_uid;
     /// Check if the entity `id` has some ENT_TYPE `type` in their inventory
     lua["entity_has_item_type"] = entity_has_item_type;
+    /// Check if the entity `id` has some ENT_TYPE `type` and `Mask` in their inventory. Set type and mask to 0 to return all inventory items.
```

Comment:
Maybe this should be more correctly `Gets all items of type and mask from entities inventory`